### PR TITLE
Remove tangram references

### PR DESCRIPTION
**Pull request by dv343**
Tangram is already deactivated because cartodb.js doesn't support it anymore, but there are still some references here, this PR fixes that (mostly).

https://github.com/CartoDB/cartodb/issues/13461

### CODE REVIEW

File: app/views/carto/builder/public/embeds/show.html.erb [hunk 1]
```diff
@@ -27,10 +27,6 @@
 <%= content_for(:js) do %>
   <%= insert_trackjs('builder-embeds') %>
 
-  <% if vector_render?(@visualization, params) %>
```
Comment (rochoa) [inline app/views/carto/builder/public/embeds/show.html.erb]: Is this the only thing that controls whether we should render with vector/Tangram or raster? This only seems to include the Tangram bundle. How do the rest of the JS components decide to go on Vector or Raster mode?

@dv343 @jesusbotella

Comment (jesusbotella) [reply app/views/carto/builder/public/embeds/show.html.erb]: That logic is handled here: https://github.com/CartoDB/cartodb/blob/master/app/controllers/visualizations_controller_helper.rb#L115-L127

I was going to remove it, as well as the tests and the feature flag, in another pull request. I thought this PR was only meant to remove the Tangram references.

Comment (jesusbotella) [reply app/views/carto/builder/public/embeds/show.html.erb]: And here too: https://github.com/CartoDB/cartodb/blob/master/app/controllers/carto/api/vizjson3_presenter.rb#L69-L71

Comment (rochoa) [reply app/views/carto/builder/public/embeds/show.html.erb]: 👍

Comment (jesusbotella) [reply app/views/carto/builder/public/embeds/show.html.erb]: Please check this PR I have just opened: https://github.com/CartoDB/cartodb/pull/13512

This handles the removal of that logic. Still WIP though.

**Pull request merged**